### PR TITLE
Make left gripper interface

### PR DESCRIPTION
**Pull request by pazeshun**
~~Works with #1654~~
Works with #1667

### CODE REVIEW

File: jsk_2016_01_baxter_apc/src/enable_gripper.cpp [hunk 1]
```diff
@@ -1,66 +1,85 @@
...
-      msg.data = false;
-      right_servo_torque_pub.publish(msg);
-      ROS_INFO("%s: Power off right gripper's servo becase no command came to right gripper in this %d minutes",
-               ros::this_node::getName().c_str(), wait_min);
+      if (g_robot_enabled && g_servo_on[*side] &&
+          (ros::Time::now() - g_last_command_time[*side]) > ros::Duration(wait_min * 60))
+      {
+        std_msgs::Bool msg;
+        msg.data = false;
+        servo_torque_pub["right"].publish(msg);
```
Comment (ban-masa) [inline jsk_2016_01_baxter_apc/src/enable_gripper.cpp]: "right" → *side

**Comment by pazeshun** [comment]
Fixed

**Comment by ban-masa** [comment]
+1

**Comment by pazeshun** [comment]
Tested in real robot with #1655, #1662, #1656, #1663. No more WIP.

**Comment by pazeshun** [comment]
Please merge this.

**Pull request merged**